lib/service/postcard_service.dart

```diff
@@ -403,6 +405,27 @@ class PostcardServiceImpl extends PostcardService {
     return tempFile;
   }
 
+  Future<File> _downloadPostcard(String tokenId) async {
+    final tempFilePath =
+        "${(await getTemporaryDirectory()).path}/Postcard/$tokenId/postcard.png";
+    final tempFile = File(tempFilePath);
+    final isFileExist = await tempFile.exists();
```

Comment:
The postcard may be change when other users stamp

Reply:
i'll fix by delete the temp file after save image. it's better solution than using different name

Reply:
You can override write file instead of delete it. Because the function `downloadPostcard`will return an empty file if delete.

Reply:
we delete the temp file after save postcard to the gallery, we don't need temp file any more at that time. see my latest commit

Reply:
currently we don't use return file of download stamp/postcard function, but should not return temp file because file in temp file may get delete at any time. we should return the file or path of the file in gallery